torchgeo/models/resnet.py

```diff
@@ -13,14 +13,64 @@
 
 # https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/transfer_classification/linear_BE_moco.py#L167
```

Comment:
I think these comments should be moved down to the s2 section. Maybe you can add a link pointing to where the S1 metrics are defined.